### PR TITLE
Doc 11127  release notes CBL 3.1.1

### PR DESCRIPTION
**Pull request by franrob-projects**
- Release notes for CBL 3.1.1 (maintenance) 
- Jira [DOC-11127](https://issues.couchbase.com/browse/DOC-11127)
- Issues are listed [here](https://issues.couchbase.com/issues/?jql=project%20%3D%20CBL%20AND%20fixVersion%20%3D%203.1.1%20and%20level%20%20%3D%20%22Public%22).

### CODE REVIEW

File: modules/android/pages/releasenotes.adoc [hunk 1]
```diff
@@ -9,7 +9,6 @@ ifdef::prerelease[:page-status: {prerelease}]
 include::partial$_set_page_context_for_android.adoc[]
 
 [#maint-latest]
-include::partial$release-notes/couchbase-mobile-android-release-note.3.1.0.adoc[]
+include::partial$release-notes/couchbase-mobile-android-release-note.3.1.1.adoc[]
 
-include::partial$release-notes/couchbase-mobile-android-release-note.3.0.5.adoc[]
-include::{root-commons}releasenotes.adoc[]
```
Comment (osfameron) [inline modules/android/pages/releasenotes.adoc:15]: why have you deleted 3.0.5 and the previous include?

Comment (osfameron) [reply modules/android/pages/releasenotes.adoc:15]: Oh I see, yes, remove 3.0.5!
But I think you want to retain include::{root-commons}releasenotes.adoc[] ?

Or is that empty?

**Review by osfameron — APPROVED** [review]
OK, a question, but I may be confusing myself, and assume you've verified the output... so 👍

File: modules/android/pages/releasenotes.adoc (hunk 1 above)
Comment (franrob-projects) [reply modules/android/pages/releasenotes.adoc:15]: We cover 3.0.x under the 3.0.x tab in the Nav

**Pull request merged**